### PR TITLE
Ui libraries corrections

### PR DESCRIPTION
**Pull request by Mohammer5**
See commits for a detailed list of changes

### CODE REVIEW

File: workshop-1/02-ui-libraries/README.md [hunk 1]
```diff
@@ -7,9 +26,43 @@ All files that you have to touch contain explanatory comments, even for code
...
+[https://codesandbox.io/s/github/dhis2/academy-web-app-dev-2020/tree/master/workshop-1/02-ui-libraries/ui-libraries-task-template](https://codesandbox.io/s/github/dhis2/academy-web-app-dev-2020/tree/master/workshop-1/02-ui-libraries/ui-libraries-task-template)
+
+It is read-only, so you need to fork it in order to work on it.
+The `@dhis2/ui` library is already installed, so you don't need to worry about
+that. You can start hacking right away.
+
+<a name="training-environment--local-dev" href=""></a>
+### If you're using your local development environment
+
+IMPORTANT NOTICE: We're not supporting local development during the webinar. So
```
Comment (amcgee) [inline workshop-1/02-ui-libraries/README.md]: `webinar` should be `workshop` or `academy`

File: workshop-1/02-ui-libraries/README.md [hunk 2]
```diff
@@ -7,9 +26,43 @@ All files that you have to touch contain explanatory comments, even for code
...
+if you're trying this and run into complications, please switch to using the
+sandbox linked to above.
+
+The webinar will introduce the code sandbox as development environment as it
+works OS independently. If you're reading this at a later time or want to use
+your local development environment, here are the steps required to get the
+template working:
```
Comment (amcgee) [inline workshop-1/02-ui-libraries/README.md]: Fine to include this, just note that this will be covered in an earlier session

**Review by amcgee — APPROVED** [review]
🎉

File: workshop-1/02-ui-libraries/README.md (hunk 1 above)
Comment (Mohammer5) [reply workshop-1/02-ui-libraries/README.md]: Renamed to workshop

File: workshop-1/02-ui-libraries/README.md (hunk 2 above)
Comment (Mohammer5) [reply workshop-1/02-ui-libraries/README.md]: I have changed the `or` to `and`, won't explain this during the presentation but will explain that the participants have to use the code sandbox in order to send us their results (I suppose you'll explain that anyways, but I think it's good to repeat that)

**Review by amcgee — APPROVED** [review]

**Pull request merged**